fv3core/testing/validation.py

```diff
@@ -114,14 +122,22 @@ def _set_nans(self, tracers: Mapping[str, Quantity]):
     return SelectivelyValidatedTracerAdvection
 
 
-def get_compute_domain(
+def get_compute_domain_k_interfaces(
     instance,
 ) -> Tuple[Tuple[int, ...], Tuple[int, ...]]:
     origin = instance.grid.compute_origin()
     domain = instance.grid.domain_shape_compute(add=(0, 0, 1))
     return origin, domain
 
 
+def get_compute_domain_2d(
```

Comment:
I'll keep the code-edit comments i wrote below for reference/feedback, but I see now that this function isn't actually used - can you delete it?